chapter04_convolutional-neural-networks/kaggle-gluon-cifar10.md

```diff
@@ -0,0 +1,412 @@
+# 实战Kaggle比赛——使用Gluon对原始图像文件分类（CIFAR-10）
+
+
+我们在[监督学习中的一章](../chapter02_supervised-learning/kaggle-gluon-kfold.md)里，以[房价预测问题](https://www.kaggle.com/c/house-prices-advanced-regression-techniques)为例，介绍了如何使用``Gluon``来实战[Kaggle比赛](https://www.kaggle.com)。社区中的很多小伙伴实践了房
+价预测问题并[分享了自己的成绩和方法](https://discuss.gluon.ai/t/topic/1039)。有小伙伴[反馈希望增加Kaggle比赛章节](https://discuss.gluon.ai/t/topic/1341)并[反馈希望多从直接与数据打交道中获得灵感](https://discuss.gluon.ai/t/topic/1519)。还有小伙伴[反馈希望提供对原始图像分类的例子](https://discuss.gluon.ai/t/topic/499)，
```

Comment:
no need to mention the discuss link here, because it aims for a book chapter

Reply:
resolved